sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1059,9 +1059,10 @@ void
 handler::ext_oneapi_graph(command_graph<graph_state::executable>& graph)
 ----
 
-|Invokes the execution of a graph. Only one instance of `graph` may be executing,
-or pending execution, at any time. Concurrent graph execution can be achieved by
-finalizing a graph in modifiable state into multiple graphs in executable state.
+|Invokes the execution of a graph. Only one instance of `graph` will
+execute at any time. If `graph` is submitted multiple times, dependencies
+are automatically added by the runtime to prevent concurrent executions of 
```

Comment:
suppose we have the below code:

h.ext_oneapi_graph(g1)   # s1: submission 1
h.ext_oneapi_graph(g1)   # s2: submission 2

for in order queue, it makes sense that dependency between s1 and s2 is automatically added by the runtime.   But for out of order queue, the runtime also adds dependency between s1 and s2 automatically? If so, it breaks the definition of out of order queue.